lib/phlex/test_helpers.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Phlex
+	module TestHelpers
+		def render_view(view, &block)
+			rendered = controller.view_context.render(view, &block)
+			Nokogiri::HTML.fragment(rendered)
+		end
+
+		def controller
+			ActionView::TestCase::TestController.new
```

Comment:
Should we memoize this?

```suggestion
			@controller ||= ActionView::TestCase::TestController.new
```